File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Silverlight/1031/BusinessApplication/Assets/Resources/ApplicationResources.cs
namespace $safeprojectname$
{
    using System;
    using System.ComponentModel;
    using System.Globalization;
    using System.Windows.Browser;

    /// <summary>
    /// Umschließt den Zugriff auf stark typisierte Ressourcenklassen, damit Steuerelementeigenschaften an Ressourcenzeichenfolgen in XAML gebunden werden können.
    /// </summary>
    public sealed class ApplicationResources
    {
        private static readonly ApplicationStrings applicationStrings = new ApplicationStrings();
        private static readonly ErrorResources errorResources = new ErrorResources();

        /// <summary>
        /// Ruft die <see cref="ApplicationStrings"/> ab.
        /// </summary>
        public ApplicationStrings Strings
        {
            get { return applicationStrings; }
        }

        /// <summary>
        /// Ruft die <see cref="ErrorResources"/> ab.
        /// </summary>
        public ErrorResources Errors
        {
            get { return errorResources; }
        }
    }
}

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Silverlight/1031/BusinessApplication/Assets/Resources/ApplicationResources.vb
Imports System
Imports System.ComponentModel
Imports System.Globalization
Imports System.Windows.Browser


''' <summary>
''' Umschließt den Zugriff auf stark typisierte Ressourcenklassen, damit Steuerelementeigenschaften an Ressourcenzeichenfolgen in XAML gebunden werden können.
''' </summary>
Public NotInheritable Class ApplicationResources
    Private Shared _applicationStrings As New ApplicationStrings()
    Private Shared _errorResources As New ErrorResources()

    ''' <summary>
    ''' Ruft die <see cref="ApplicationStrings"/> ab.
    ''' </summary>
    Public ReadOnly Property Strings() As ApplicationStrings
        Get
            Return _applicationStrings
        End Get
    End Property

    ''' <summary>
    ''' Ruft die <see cref="ErrorResources"/> ab.
    ''' </summary>
    Public ReadOnly Property Errors() As ErrorResources
        Get
            Return _errorResources
        End Get
    End Property
End Class
